### DISCUSSION TITLE
Possibility to have extended breaks every X regular breaks

### DISCUSSION THREAD

**Discussion by marianafcosta** [tom-james-watson/breaktimer-app#180] [Ideas]
I've been playing around with the app a bit and added the possibility of alternating between short and long breaks (e.g. to follow the Pomodoro method). Would this be something that other people could find useful?

By the way, lovely app! Simple and gets the job done

**Comment by mustdobetter** [comment]
I was thinking along the same lines. I find it helpful to take a short eye break (~20 seconds) every 20 minutes and a longer break every hour or so. So either explicit support for short and long breaks, or for ultimate flexibility, an extensible list of configurable breaks.

**Comment by Artaherzadeh** [comment]
This is a nice feature. Stretchly app also has this.
